uint8 id
float32 agl_0
float32 agl_1
float32 agl_2